integration/samples.sh

```diff
@@ -1,37 +1,60 @@
-#!/bin/bash
-set -e
-
-FIND_COMMAND=""
-
-if [[ $OSTYPE == "win32" ]]; then
-    FIND_COMMAND="dir /b /o:n /ad"
-else
-    FIND_COMMAND="find ./integration/samples/* -maxdepth 0 -type d"
-fi
-
-for D in $FIND_COMMAND
-do
-    P_JS=${D}/ng-package.js
-    P_JSON=${D}/ng-package.json
-    P_API=${D}/ng-packagr-api.js
-    P=${D}/package.json
-
-    if [ -f $P_API ]; then
-        echo "Building sample ${P_API}..."
-        node ${P_API}
-    elif [ -f $P_JS ]; then
-        echo "Building sample ${P_JS}..."
-        node dist/cli/main.js -p ${P_JS}
-    elif [ -f $P_JSON ]; then
-        echo "Building sample ${P_JSON}..."
-        node dist/cli/main.js -p ${P_JSON}
-    elif [ -f $P ]; then
-        echo "Building sample ${P}..."
-        if [ $P='failures' ]; then
```

Comment:
FYI: This was also failing (as I mentioned in https://github.com/ng-packagr/ng-packagr/pull/1845#discussion_r563229441). 

In the current PR we switched to a `case`, since that's the only POSIX compatible way to do it (besides grep, awk and what not).
```

    # All integration samples starting with "fail" are expected to fail.
    EXPECTED_TO_FAIL=false
    case "${DIR_NAME}" in
        "fail"*)
            printf "NOTE: This next integration sample build is expected to fail!\n"
            EXPECTED_TO_FAIL=true
            ;;
    esac
```